version/info_default.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !go1.18
```

Comment:
Is this future proof for >= 1.19?

Reply:
It looks like it is. I tested this by adding a `test_go118.go` to an existing project:

```
//go:build go1.18

package main

import "log"

func init() {
        log.Println("1.18")
}
```

Running this with Go 1.19 printed the line. Flipping it to `//go:build !go1.18` made it go away when compiled/run with Go 1.19.